functions/src/admin/AdminPaymentTaskProcessor.ts

```diff
@@ -77,6 +81,64 @@ export class AdminPaymentTaskProcessor {
 		return `Created ${paymentsCreated} payments`;
 	};
 
+	private sendNotifications = async (recipientDocs: QueryDocumentSnapshot<Recipient>[]) => {
+		let newNotificationsSent = 0;
+		let existingNotifications = 0;
+		const now = moment();
+
+		for (const recipientDoc of recipientDocs) {
+			if (recipientDoc.data().test_recipient) continue;
+
+			const paymentDocRef = this.firestoreAdmin.doc<Payment>(
+				`${RECIPIENT_FIRESTORE_PATH}/${recipientDoc.id}/${PAYMENT_FIRESTORE_PATH}`,
+				now.format('YYYY-MM')
+			);
+
+			if ((await paymentDocRef.get()).exists) {
+				const paymentDocSnap = await paymentDocRef.get();
+				const payment: Payment = paymentDocSnap.data() as Payment;
+				if (!payment.message) {
+					const recipient: Recipient = recipientDoc.data();
+					const [messageSid, _messageStatus, messageContent] = await sendSms({
+						messageRecipientPhone: '+' + recipient.mobile_money_phone.phone,
+						messageContext: {
+							content: 'This is a Test SMS',
```

Comment:
Change the content?

Reply:
Here a first proposition (the length can be up to [160 characters](https://www.twilio.com/docs/glossary/what-sms-character-limit)): 

EN: Dear {Calling Name, _else_ First Name}, you should receive your monthly Social Income ({SLE 500}) shortly. Thanks you for confirming the payment immediately.

KRIO: Diya {Calling Name, _else_ First Name}, yu fɔ gɛt yu Social Income ({SLE 500}) we yu kin gɛt ɛvri mɔnt shɔt tɛm. Tɛnki fɔ kɔnfɔm di pe wantɛm wantɛm.

Explanation: 

- I would start with a generic one (without mentioning the channel they normally use to confirm). This can be in a second step, we would need the data of their preferred channel first in the DB (is coming with the new architecture of the number fields). 
- {...} are variables, where as the amount SLE 500 could be taken from the `payment.ts` -> `PAYMENT_AMOUNT`